settings.gradle.kts

```diff
@@ -25,7 +32,16 @@ reckon {
 }
 
 dependencyResolutionManagement {
+    repositoriesMode = RepositoriesMode.FAIL_ON_PROJECT_REPOS
     repositories {
+        google {
+            content {
+                includeGroupByRegex("com\\.android.*")
+                includeGroupByRegex("com\\.google.*")
+                includeGroupByRegex("androidx.*")
+                includeGroupByRegex("org\\.chromium.*")
```

Comment:
I took the liberty to make sure we download the libraries from the place we are expecting them to be.

Reply:
Well in build-logic we don't use chromium, but it is also true for `androidx` and I kept it 🤔. Should we match or should we try to extract this into a File that is loaded on both side?

Reply:
I'm not sure which option is the best, but don't like inconsistency. If there's a good reason to have them be different that's fine but maybe add a one line comment about it? Honestly not quite sure how this interacts with the one in build-logic/settings.gradle.kts.

Reply:
I've played with how I can share this and I'm not a big fan of the solution, I had to create another project and add another includeBuild. 
![image](https://github.com/user-attachments/assets/45a63953-afea-4eb8-bfce-db44c21a6e55)

It adds a third layer in the equation and a new settings.gradle.kts (that is only referencing `mavenCentral`). Usage of ktlint from `libs` is not possible or I need to duplicate the line `id("dev.panuszewski.typesafe-conventions") version "0.4.1"` somewhere.

Lemme know what do you prefer. From my point of view after some though I don't mind having a different `repositories` settings in the two project. It doesn't leak from one to the other. For instance jitpack is not available in `build-logic`.

Reply:
... another project seems like a lot of complication for aligning one thing that will rarely if ever be touched.

Let's keep it as suggested in the PR for now 😅